src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -48,6 +51,19 @@ public function isGranted(mixed $role, mixed $object = null, ?string $field = nu
         }
     }
 
+    public function isGrantedForUser(UserInterface $user, mixed $attribute, mixed $subject = null, ?string $field = null): bool
+    {
+        if (null === $this->userSecurityChecker) {
```

Comment:
I know this was consistent with `isGranted()` but  let's align to our current coding style
```suggestion
        if (!$this->userSecurityChecker) {
```

Reply:
Updated for both comments.